jarviscli/packages/fb.py

```diff
@@ -0,0 +1,50 @@
+import getpass
+import os
+from time import sleep
+from selenium import webdriver
+
+
+def get_details():
+    file = 'password.txt'
+
+    # retrive password if saved previously
+
+    if os.path.isfile(file):
+        with open(file, 'r') as f:
+            usr = f.readline()
+            pwd = f.readline()
+            usr = usr.strip('\n')
+
+    # passwords are not saved
+
+    else:
+        usr = raw_input('Enter Email Id: ')
+        pwd = getpass.getpass('Enter Password: ')
+        choice = raw_input('Do you want to save the id and password (y/n): ')
+        if choice == 'y' or choice == 'Y':
+            with open(file, 'w') as f:
+                f.write(usr)
+                f.write('\n')
+                f.write(pwd)
+    return usr, pwd
+
+
+def fb_login(self):
+    usr, pwd = get_details()
+
+    driver = webdriver.Chrome()
```

Comment:
the thing is I have chromedriver installed so my pc pass this script anyways. How can I test like someone else is testing.